include/drivers/spi.h

```diff
@@ -162,6 +162,11 @@ struct spi_cs_control {
  *
  * @note Only cs_hold and lock_on can be changed between consecutive
```

Comment:
See this comment.

Drivers that do compare the config pointer should only config on these 2 fields, if the config if the same then.

Reply:
That comment should probably be removed, then, because most drivers use the return value to bypass all configuration.

Reply:
Drivers should be fixed instead.